### PR TITLE
Update list of known bugs on Raspberry Pi

### PR DESCRIPTION
**Pull request by ggardet**
VR: <s>https://openqa.opensuse.org/tests/1838577#step/journal_check/10</s> https://openqa.opensuse.org/t1838702

### CODE REVIEW

**Comment by mloviska** [comment]
LGTM, could you please add https://openqa.opensuse.org/tests/1836456#step/journal_check/13 ?

**Comment by ggardet** [comment]
> LGTM, could you please add https://openqa.opensuse.org/tests/1836456#step/journal_check/13 ?

Done.

**Review by mloviska — APPROVED** [review]
Let's wait for the VR. Thanks!

File: data/journal_check/bug_refs.json [hunk 1]
```diff
@@ -20,10 +20,17 @@
...
             "sle-micro": [ "5.0", "5.1" ],
-            "opensuse": ["15.3"]
+            "opensuse": ["Tumbleweed", "15.3", "15.2"]
         },
         "type": "bug"
     },
+    "bsc#1188270": {
+        "description": "lan78xx 1-1.1.1:1.0 eth0: No phy led trigger registered for speed\\(-1\\)",
+        "products": {
+            "opensuse": ["Tumbleweed"]
```
Comment (jlausuch) [inline data/journal_check/bug_refs.json:30]: we need to add 15.3 here as well https://openqa.opensuse.org/tests/1840709#step/journal_check/12

**Pull request merged**